advanced/javascript/run-javascript.md

```diff
@@ -43,3 +43,58 @@ For very simple computations (like the one above), creating a new file might be
 {% content-ref url="../../api-reference/commands/evalscript.md" %}
 [evalscript.md](../../api-reference/commands/evalscript.md)
 {% endcontent-ref %}
+
+### Inbuilt functions & properties
+
+Whilst the JavaScript environment is limited, there are a few inbuilt things that can be used:
+
+#### maestro object
+
+The maestro object contains the following properties:
+
+| Field Name    | Value                    |
+| ----------    | ------------------------ |
+| `copiedText`  | Results of the [copyTextFrom](../../api-reference/commands/copytextfrom.md) command. See [Access element text](./access-element-text.md) |
+| `platform`    | The platform the test is running on. Either `ios` or `android` |
```

Comment:
What will the value be if we're running on the [experimental webdriver](https://maestro.mobile.dev/advanced/experimental/maestro-webdriver)?